src/main/resources/db/migration/V5.11.0.1__hdc-3b.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE usernotification ADD COLUMN IF NOT EXISTS additionalinfo VARCHAR;
```

Comment:
Please note that we already have a flyway script using V5.11.0.1 in the develop branch (V5.11.0.1__8605-support-archival-status.sql)...